addons/web/static/src/webclient/settings_form_view/settings_form_controller.js

```diff
@@ -61,7 +61,6 @@ export class SettingsFormController extends formView.Controller {
                 this.dialogService.add(SettingsConfirmationDialog, {
                     body: message,
                     confirm: async () => {
-                        await this.model.root.save({ stayInEdition: true });
```

Comment:
@kebeclibre do you agree with this?
@jpp-odoo does it cause any issue? We should maybe remove it in master...

Reply:
@aab-odoo The issue here is that `_save()` will call `onClickViewButton` that will call `beforeExecuteActionButton` again. And as we removed the id after the save, we will call twice the `create`.
I removed to avoid having an extra rpc.

Reply:
ok